x-pack/plugins/reporting/server/browsers/phantom/driver/index.js

```diff
@@ -18,24 +18,37 @@ export function PhantomDriver({ page, browser, zoom, logger }) {
     if (page === false || browser === false) throw new Error('Phantom instance is closed');
   };
 
-  const configurePage = (pageOptions) => {
+  const configurePage = () => {
     const RESOURCE_TIMEOUT = 5000;
     return fromCallback(cb => page.set('resourceTimeout', RESOURCE_TIMEOUT, cb))
       .then(() => {
         if (zoom) return fromCallback(cb => page.set('zoomFactor', zoom, cb));
-      })
-      .then(() => {
-        if (pageOptions.headers) return fromCallback(cb => page.set('customHeaders', pageOptions.headers, cb));
       });
   };
 
   return {
     open(url, pageOptions) {
       validateInstance();
 
-      return configurePage(pageOptions)
+      return configurePage()
         .then(() => logger.debug('Configured page'))
         .then(() => fromCallback(cb => page.open(url, cb)))
+        .then(async (status) => {
+          const { sessionCookie } =  pageOptions;
+          if (sessionCookie) {
+            await fromCallback(cb => page.clearCookies(cb));
+            await fromCallback(cb => page.addCookie({
```

Comment:
Is it worth adding a comment here why `SameSite` isn't used here, but is for Chromium?